packages/tasit-sdk/src/index.test.js

```diff
@@ -0,0 +1,9 @@
+import { Account } from "./index.js";
```

Comment:
@marcelomorgado I usually leave off the `.js` suffix. Just a style thing though - either way works.